tools/syz-cover/syz-cover.go

```diff
@@ -127,6 +128,37 @@ func main() {
 		tool.Fail(err)
 	}
 	fn += ".html"
+	log.Logf(0, "write to %v", fn)
```

Comment:
Could you please elaborate on what is happening below and why it was necessary?

Also, whatever it is, I'd keep it optional and only do that if a flag is passed -- there's no guarantee that `xdg-open` is present on a system where `syz-cover` is run.

Reply:
not related to this PR, so removed